react-training/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+
+import LoginForm from './LoginForm';
+
+class Login extends Component {
+  state = {
+    isLoggedIn: false
```

Comment:
In this component u must render only the login layout. The isLogged behavior shouldn't belong to this component, but don't care. In another pr, when you have to do the functionality of keeping the session active, we will do this in another way